os_net_config/impl_nmstate.py

```diff
@@ -2386,6 +2413,12 @@ def apply(self, cleanup=False, activate=True, config_rules_dns=True):
                 self.rollback_to_initial_settings()
                 raise os_net_config.ConfigurationError(message)
 
+            try:
+                if rules_applied:
+                    common.write_yaml_config(CONFIG_RULES_FILE, rules_applied)
+            except NameError:
```

Comment:
no_rules = self.set_rules(rules_data=[])
common.write_yaml_config(CONFIG_RULES_FILE, no_rules)

and remove `rules_template`